packages/server/package.json

```diff
@@ -12,8 +12,6 @@
     "citest": "npm run test"
   },
   "dependencies": {
-    "common": "0.0.1",
```

Comment:
Are these unused dependencies as a result of your change, or a separate thing?  If the latter, should do it in a separate PR.

Reply:
As a result of the change. I cut these out because I wasn't sure if they were causing this issue somehow.